irr/include/IrrlichtDevice.h

```diff
@@ -342,6 +343,20 @@ class IrrlichtDevice : public virtual IReferenceCounted
 	{
 		return video::isDriverSupported(driver);
 	}
+
+#if defined(_IRR_COMPILE_WITH_SDL_DEVICE_) || USE_SDL2
```

Comment:
i dont think this ifdef is needed

Reply:
Anyway I dropped this ifdef in e13e1bc92603b2962170ecb7c050b2fd4af1951c.